### DISCUSSION TITLE
How to change the stop loss (SL)?

### DISCUSSION THREAD

**Discussion by stelioszach03** [EA31337/EA31337#394] [Q&A]
hello i want to ask how can i change the stop loss.
Thank You!!

**Comment by kenorb** [comment]
By default stop losses are dynamic and are automatically managed by strategies and risk margin (which you can set in %). In Advanced/Rider, you can change stop strategies which will affect how the stop losses are calculated.
If you're looking for fixed stop losses, you can use _Close loss/profit_ param, however they're stealth stops (which means they're not displayed as TP/SL, but being tracked by EA and orders are going to close automatically when the condition is met).
